shells/dev/app/SuspenseTree/index.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import React, { Suspense, useState } from 'react';
+
+function SuspenseTree() {
+  return (
+    <>
+      <h1>Suspense</h1>
+      <Suspense fallback={<h2>Loading outer</h2>}>
+        <Parent />
+      </Suspense>
+    </>
+  );
+}
+
+function Never() {
+  throw new Promise(resolve => {});
+}
+
+function Parent() {
+  return (
+    <div>
+      <Suspense fallback={<h3>Loading inner 1</h3>}>
+        <Child>Hello</Child>
+      </Suspense>
+      <Suspense fallback={<h3>Loading inner 2</h3>}>
+        <Child>World</Child>
+      </Suspense>
+      <Suspense fallback={<h3>This will never load</h3>}>
+        <Never />
+      </Suspense>
+      <LoadLater />
+    </div>
+  );
+}
+
+function LoadLater() {
+  const [loadChild, setLoadChild] = useState(0);
+  return (
+    <Suspense
```

Comment:
Selecting this element in the DevTools tree and clicking the load/reload button causes an error:
```
Uncaught TypeError: Cannot read property 'parentID' of undefined
    at _loop (devtools.js:13578)
    at Bridge.<anonymous> (devtools.js:13649)
    at Bridge.emit (devtools.js:1621)
    at devtools.js:12785
    at devtools.js:21985
```

Reply:
Actually this happens when you toggle via the inspector too (makes sense). Seems like a more important thing to fix.

![toggle-bug-Kapture 2019-04-04 at 13 45 36](https://user-images.githubusercontent.com/29597/55587356-064ec780-56e0-11e9-9e16-955c9206f4c8.gif)